app/views/distributions/_distribution.html.erb

```diff
@@ -0,0 +1,135 @@
+<header class="download-home-header jumbotron jumbotron-fluid bg-<%= @colour %> text-center pb-0">
+  <div class="container">
+    <div class="page-heading pb-5">
+      <% if @distro_type == "leap" %>
+        <%= leap_image_tag alt: "Leap Logo", class: "animated jackInTheBox", width: "256", height: "256" %>
```

Comment:
what about adding width and height to the css of a class?

Reply:
Good point